rusk-wallet/src/main.rs

```diff
@@ -272,6 +272,14 @@ async fn rusk_uds(socket_path: String) -> Result<Rusk, Error> {
 
 #[tokio::main]
 async fn main() -> Result<(), Error> {
+    if let Err(err) = exec().await {
+        println!("{}", err);
+        println!("Terminating");
```

Comment:
I would remove the "Terminating" line, since it also prints when it returns successfully. Just a nit though.